src/handlers/multi-class-value-handler.ts

```diff
@@ -0,0 +1,57 @@
+import { ValueHandler } from '../types/value-handler.interface';
+import { PropertyDto } from '../types/property-dto.interface';
+import { ExactValue, ClassType } from '../types/fixture-options.type';
+import { MultiClass } from '../types/fixture-options.type';
+import { ClassProcessor } from '../class-processor';
+import { PrimitiveHandlerAbstract } from './primitive-handler-abstract';
+
+import FakerStatic = Faker.FakerStatic;
+
+export class MultiClassValueHandler<P extends MultiClass>
+  extends PrimitiveHandlerAbstract<P>
+  implements ValueHandler<P> {
+  public constructor(
+    protected readonly faker: FakerStatic,
+    protected readonly classProcessor: ClassProcessor<ClassType>
+  ) {
+    super(faker);
+  }
+
+  public static isTypeValue(propertyDto: PropertyDto<MultiClass>): boolean {
+    const { value = '' } = propertyDto;
+
+    return Object.prototype.hasOwnProperty.call(value, 'type');
+  }
+
+  public shouldHandle(propertyDto: PropertyDto<P>): boolean {
+    return propertyDto.type === 'object' && MultiClassValueHandler.isTypeValue(propertyDto);
+  }
+
+  public produceValue<T>(propertyDto: PropertyDto<P>): any {
+    const { value } = propertyDto;
+
+    if (value === null) {
+      return value;
+    }
+
+    const { count, type } = value;
+
+    if (PrimitiveHandlerAbstract.PRIMITIVES.includes(type.name)) {
```

Comment:
This part is wired.
why do we have primitive handling in MultiClassValueHandler ?